apiserver/plane/app/views/workspace.py

```diff
@@ -1238,6 +1241,66 @@ def get(self, request, slug, user_id):
         )
 
 
+class ExportWorkspaceUserActivityEndpoint(BaseAPIView):
+    permission_classes = [
+        WorkspaceEntityPermission,
```

Comment:
Only admins and members should be allowed and also change the method to `POST`

Reply:
Changed the permission layer, only the admin and members can download the CSV